### PR TITLE
Package Documentation + Migrated Bundled Data

### PR DESCRIPTION
**Pull request by mattssca**
This PR adds missing bundled data (previously in main GAMBLR) to this repo. In addition, dataset documentation has also been developed and documentation resources compiled (addressing issue #9).

### CODE REVIEW

File: R/data.R [hunk 1]
```diff
@@ -1,510 +1,877 @@
...
-#' lymphoma studies have been curated and provided with this package.
+#' @format ## `gene_blacklist`
+#' A tibble with 291 rows.
+#' \describe{
+#'   \item{Gene}{Genes symbols in Hugo format.}
+#' }
+"gene_blacklist"
+
+
+#' Grande et al. MAF.
```
Comment (Kdreval) [inline R/data.R]: The grande_maf should be deprecated now, since we have that sample_data now 😄

**Review by Kdreval — COMMENTED** [review]
Thanks Adam!
We should also here:
1. Migrate all raw data from the GAMBLR/inst
2. Migrate all the code that generates these rda from the GAMBLR/data-raw/DATASET.R
3. Just checking with you there is `lymphoma_genes` object - don't see it so maybe I have overlooked it?

File: R/data.R (hunk 1 above)
Comment (mattssca) [reply R/data.R]: For sure, I just migrated all the datasets in GAMBLR main repo, so we would only have to remove any unwanted objects (like this one). I'll address this in a commit along with the other items you pointed out. Thanks!

File: data-raw/DATASET.R [hunk 2]
```diff
@@ -150,5 +150,251 @@ hg38_ashm_regions <- read.delim(
...
+lymphoma_genes = dplyr::select(lymphoma_genes,-entrezgene_id) %>% group_by(Gene,ensembl_gene_id) %>% slice_head() %>% ungroup()
+
+#DLBCL_curated_genes = lymphoma_genes %>% dplyr::filter(DLBCL==TRUE) %>% pull(Gene)
+
+#lymphoma_genes_comprehensive$curated = FALSE
+#lymphoma_genes_comprehensive[lymphoma_genes_comprehensive$Gene %in% DLBCL_curated_genes,]$curated = TRUE
+#lymphoma_genes_comprehensive = dplyr::filter(lymphoma_genes_comprehensive,Reddy==TRUE | Chapuy == TRUE | LymphGen == TRUE | curated == TRUE)
+#lymphoma_genes_comprehensive$other_support = ""
+
+# Example SSM data from Grande et. al, 2019
```
Comment (Kdreval) [inline data-raw/DATASET.R:368]: This section can probably be retired

File: inst/extdata/blood8871418-suppl2-ssm.csv
Comment (Kdreval) [inline inst/extdata/blood8871418-suppl2-ssm.csv:1]: This can probably be retired :older_man:

File: DESCRIPTION [hunk 3]
```diff
@@ -20,11 +20,15 @@ Description: The package contains manually curated data for the genomic Analysis
...
 Depends:
     R (>= 2.10)
 LazyData: true
 Imports: 
     dplyr,
     ggplot2,
-    tibble
+    tibble,
+    tidyr
+Remotes:
```
Comment (Kdreval) [inline DESCRIPTION:32]: Which helpers are used here? I think it also needs the GAMBLR.helpers to be listed in the `Imports` section

File: inst/extdata/blood8871418-suppl2-ssm.csv
Comment (mattssca) [reply inst/extdata/blood8871418-suppl2-ssm.csv:1]: Indeed, thanks for pointing it out.

**Review by vladimirsouza — APPROVED** [review]

**Pull request merged**